src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+
+const Todo = props => { 
+    return (
+        <h2
+        onClick={() => props.changeFinished(props.todoOnProp.id)}
+        className={`eachListed ${props.todoOnProp.completed}`}>
```

Comment:
I like to see how semantic your names are, you'll slip into your own convention as time goes on but it's awesome to see that you're actually thinking about what all these values are